packages/identity/src/identity.js

```diff
@@ -6,8 +6,8 @@ import { obfuscateUserId } from './userId';
 import { createUserSecretB64 } from './userSecret';
 
 type PermanentIdentityTarget = 'user';
-type SecretProvisionalIdentityTarget = 'email';
-type PublicProvisionalIdentityTarget = 'email' | 'hashed_email';
+type SecretProvisionalIdentityTarget = 'email' | 'phone_number';
+type PublicProvisionalIdentityTarget = 'email' | 'hashed_email' | 'phone_number';
```

Comment:
hashed_phone_number?

Reply:
Sorry, again this is a feature branch so you are looking through the history =)  
This was a decision that was taken later (and the TEP updated), so it is done in another commit.